src/test_driver/linux-cirque/SubscriptionResumptionCapacityTest.py

```diff
@@ -52,7 +52,7 @@
 TEST_EXTPANID = "fedcba9876543210"
 TEST_DISCRIMINATOR = 3840
 MATTER_DEVELOPMENT_PAA_ROOT_CERTS = "credentials/development/paa-root-certs"
-TEST_END_DEVICE_APP = "lit-icd-app"
+TEST_END_DEVICE_APP = "all-clusters-app"
```

Comment:
@wqx6 could you review this one, as an author? It looks like this and the `SubscriptionResumptionTest` tested subscription resumption with lit-icd-app that actually has `chip_subscription_timeout_resumption = false`, so it should fail. The reason why it didn't fail was that cirque test wasn't importing args.gni from linux lit-icd-app. Now I started to import it and the tests failed.

I don't see much sense in testing subscription resumption with lit-icd-app, so I proposed changing to all-clusters-app, but maybe I don't see a big picture and actually there was a purpose to test it like this.

Reply:
Anyway, for all-clusters-app it fails as well, so maybe it was not a reason.

Reply:
I agree that the all-clusters should be good.